crates/re_viewer/src/ui/welcome_screen/example_page.rs

```diff
@@ -158,6 +158,7 @@ fn load_file_size(egui_ctx: &egui::Context, url: String) -> Promise<Option<u64>>
                         response.status,
                         response.status_text
                     );
+                    sender.send(None);
```

Comment:
```suggestion
                    // TODO(EmbarkStudios/poll-promise#25): if you drop the `Sender` without sending
                    // a result, the `Promise` will panic when polled.
                    // See <https://github.com/EmbarkStudios/poll-promise/issues/25>.
                    sender.send(None);

Reply:
This is a not a TODO - we should always send something, otherwise the promise has nothing to resolve to.